sympy/codegen/tests/test_cnodes.py

```diff
@@ -66,7 +66,7 @@ def test_sizeof():
     assert ccode(sz) == 'sizeof(%s)' % typename
     assert sz.func(*sz.args) == sz
     assert not sz.is_Atom
-    assert all(atom == typename for atom in sz.atoms())
+    assert sz.atoms() == set([String('unsigned int'), String('sizeof')])
```

Comment:
You can create a set directly rather than creating a list and converting it to a set e.g. `{String(...), String(...)}`